clients/data_types.go

```diff
@@ -131,3 +148,8 @@ type CreateBaremetalInfo struct {
 	BaremetalSpecs []CloudProperty `json:"baremetal_specs"`
 	Deployment     string          `json:"deployment"`
 }
+
+// deployment
+type Deployment struct {
+	Name string `yaml:"name"`
```

Comment:
The Deployment type is used when to unmarshal the yaml file, so I specified it with 'yaml'.(seems 'json' also works here)